src/containers/ReviewProblemStepsModal/components/ReviewProblemStepsContent/components/ResponsesList/messages.js

```diff
@@ -0,0 +1,21 @@
+import { defineMessages } from '@edx/frontend-platform/i18n';
+
+const messages = defineMessages({
+  responsesDetailListTitle: {
+    id: 'ora-grading.ReviewProblemStepsContent.responsesDetailListTitle',
+    defaultMessage: 'Responses',
+    description: 'ORA title for response list',
+  },
+  hideRubric: {
+    id: 'ora-grading.ReviewProblemStepsContent.hideRubric',
+    defaultMessage: 'Hide Rubric',
+    description: 'Review pane action bar Hide Rubric button text',
+  },
+  showRubric: {
+    id: 'ora-grading.ReviewProblemStepsContent.showRubric',
+    defaultMessage: 'Show Rubric',
```

Comment:
```suggestion
    defaultMessage: 'Show rubric',
```
If we use capitalization elsewhere then we should do it here, but if we don't then we should keep it that way for consistency